### DISCUSSION TITLE
problems with queryRaw

### DISCUSSION THREAD

**Discussion by orcioly** [prisma/prisma#20511] [Q&A]
### Question

I'm developing an API where the database is MySQL and has no relationships and in this case I'm using Prisma's queryRaw.
I tested the code below directly in the bank and it works 100% but in the code it doesn't work and doesn't show errors, Insomnia just keeps processing and doesn't finish. I believe that the JSON_OBJECT is not being recognized in the code, but it works directly in the database running this script.

I put it in a try catch for testing and the console gives an error message: "FUNCTION does not exist in the bank". But if I run the same code directly on the bank it works without problems.

Follow the code below:

```js
const voucher = await prismaClient.$queryRaw<TrackingVoucher>`
  SELECT 
      vt.id,
      vt.transactionId,
      vt.referenceId,
      vt.status_venda,
      vt.description, vt.name,
      tv.email_comprador,
  JSON_OBJECT(
      'id', te.id,
      'nome_evento', te.nome_evento
  ) AS event
  FROM
    venda_transacoes AS vt
  JOIN
    tracking_vouchers AS tv ON tv.superpnr = vt.referenceId
  INNER JOIN
    tracking_eventos AS te ON te.id = tv.id_evento
  WHERE
    vt.referenceId = ${input.superpnr} AND tv.email_comprador = ${input.email_comprador}
`;
```

Could someone help me?

### Information about Prisma Schema, Client Queries and Environment (optional)

```js
// Add your schema.prisma and relevant query code
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @orcioly 👋 

Can you please clarify what you mean by `bank`? What version of MySQL server are you using with Prisma? Can you please provide the output of this statement `SELECT VERSION()` from your MySQL server?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
